documentation/assemblies/upgrading/assembly-upgrade-cluster-operator.adoc

```diff
@@ -56,4 +50,45 @@ kubectl get kafka <kafka_cluster_name> -n <namespace> -o jsonpath='{.status.cond
 
 Replace <kafka_cluster_name> with the name of your Kafka cluster and <namespace> with the Kubernetes namespace where the pod is running.
 
+== Migrating from using the Bidirectional Topic Operator
+When deploying the Topic Operator to manage topics, the Cluster Operator enables unidirectional topic management.
+This means that the Topic Operator only manages Kafka topics associated with `KafkaTopic` resources and does not interfere with topics managed independently within the Kafka cluster.
+
+Previously, the Topic Operator worked in bidirectional mode, which meant it could also perform operations on topics within the Kafka cluster.
+If you are switching from a version of Strimzi that uses the Bidirectional Topic Operator, after upgrading the Cluster Operator, perform some cleanup tasks to delete the internal topics that were used by the operator. 
+
+For the `strimzi-store-topic` and `strimzi-topic-operator` topics, delete the resources that were used to manage them:
+
+.Deleting internal topics used by the operator
+[source,shell,subs=+quotes]
+----
+kubectl delete $(kubectl get kt -n <namespace_name> -o name | grep strimzi-store-topic) \
+  && kubectl delete $(kubectl get kt -n <namespace_name> -o name | grep strimzi-topic-operator)
+----
+
+For the internal topics for storing consumer offsets and transaction states, `consumer-offsets` and `transaction-state`, you want to retain them in Kafka, but you don't want them to be managed by the Topic Operator.
+
+Discontinue their management before deleting their resources.
+Annotating the `KafkaTopic` resources with `strimzi.io/managed="false"` indicates that the Topic Operator should no longer manage those topics: 
+
+.Discontinuing management of internal topics
+[source,shell,subs=+quotes]
+----
+kubectl annotate $(kubectl get kt -n <namespace_name> -o name | grep consumer-offsets) strimzi.io/managed="false" \
+  && kubectl annotate $(kubectl get kt -n <namespace_name> -o name | grep transaction-state) strimzi.io/managed="false"
+----
+
+Having discontinued their management, delete the `KafkaTopic` resources:
+
+.Deleting the resources for managing internal topics
+[source,shell,subs=+quotes]
+----
+kubectl delete $(kubectl get kt -n <namespace_name> -o name | grep consumer-offsets) \
+  && kubectl delete $(kubectl get kt -n <namespace_name> -o name | grep transaction-state)
+----
```

Comment:
Well, the use case in the link is a bit different. It's about changing a resource name without deleting the topic in Kafka, so I guess it would be confusing.

Reply:
Yeah, I thought it different enough to add the specific instructions here.

Reply:
I guess you have to do it completely then ... including the waiting for the reconciliation etc.

Reply:
I've added a status check to the instructions, but linked back to the example so we don't need to repeat it